github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
Potential Bug: The version of an indirect dependency "github.com/golang/protobuf" has been changed from "v1.5.2" to "v1.5.3". Please confirm that this will not cause any compatibility or functionality issues.